app/views/candidate_interface/decisions/decline_offer.html.erb

```diff
@@ -10,9 +10,9 @@
     <%= render(CandidateInterface::OfferReviewComponent.new(course_choice: @application_choice)) %>
 
     <% if single_application_choice? %>
-      <p class="govuk-body">If you decline this offer you can still apply for more courses this year.</p>
+      <p class="govuk-body">If you decline this offer you can still apply again.</p>
```

Comment:
Elsewhere we've used the line "you can still apply for courses that start this academic year" as candidates were often unaware this was the case, and sometimes assumed "apply again" meant next year... 😬  Possibly that's a bit too much on this decline-offer flow though?

Reply:
'Apply for more courses this year' (if it means academic or recruitment year) isn't always correct. For example if they submitted their application after the Apply 2 deadline, they actually have to wait for the next recruitment year. Since I couldn't see any of this logic included here, figured best just to keep it simple.

Reply:
Ah yes, I forgot about that state - well remembered! 🤦🏻‍♂️